core/src/main/resources/hudson/slaves/JNLPLauncher/main_ru.properties

```diff
@@ -20,6 +20,6 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Connected\ via\ JNLP\ agent.=Подключен через JNLP агента.
-Run\ from\ agent\ command\ line\:=Выполнить в коммандной строке подчиненного узла
-launch\ agent=запустить агента
+Connected\ via\ JNLP\ agent.=Подключен через JNLP агент.
+Run\ from\ agent\ command\ line\:=Выполнить в командной строке агента
+launch\ agent=запустить агент
```

Comment:
Just to double check, don't we need upper-cased first latter?

Reply:
it's lowercase in every other language so probably a link in the middle of text - but I tried looking for it and actually it might just be unused now?

Reply:
Looks like a leftover translation from https://github.com/jenkinsci/jenkins/pull/6543/files#diff-731cfabfa254305495da8a5d54f327e47d6229868df1c286b644500a998faa85L44